src/Homepage/Homepage.js

```diff
@@ -1,13 +1,18 @@
 import React from 'react'
+import styled from 'styled-components'
+import BulletList from '../common/BulletList'
 import Button from '../common/Button'
-import { Link } from 'react-router-dom'
+import FloatingButtonContainer from '../common/FloatingButtonContainer'
+import HeadlineUnderline from '../common/HeadlineUnderline'
 
 export default function () {
   return (
     <>
       <img alt="" src="/img/mirror.svg" />
```

Comment:
Hier fehlt ein Alt Attribut

Reply:
As long as the svg doesn't have an essential meaning to understand the page, there is no need to describe the image.